manifests/latexml.pp

```diff
@@ -0,0 +1,88 @@
+# == Class: drmf::latexml==
+class drmf::latexml(
+#   $M2_HOME = '/usr/share/maven',
+   $latexmlPort = 9999,
+   $customInputs = '/semantic-macros/DRMF',
+#   $tomcatPassword = 'vagrant'
+)  {
+  package { [
+#     'apache2',
+#     'maven',
+#     'tomcat7',
+#     'tomcat7-admin'
+  ]:
+  }
+  file { '/vagrant/srv/latexml':
+    ensure => 'directory',
+  }
+  git::clone { 'latexml':
+    remote    => 'https://github.com/brucemiller/LaTeXML.git',
+    directory => '/vagrant/srv/latexml/LaTeXML',
+    require   => File['/vagrant/srv/latexml/']
+  }
+  git::clone { 'ltxpsgi':
+    remote    => 'https://github.com/dginev/LaTeXML-Plugin-ltxpsgi.git',
+    directory => '/vagrant/srv/latexml/ltxpsgi',
+  }
+  git::clone { 'search':
+    remote    => 'https://github.com/KWARC/LaTeXML-Plugin-MathWebSearch.git',
+    directory => '/vagrant/srv/latexml/search',
+  }
+ 
+
+  exec { 'latexml prerequisites':
+    command => '/usr/bin/apt-get install libarchive-zip-perl libfile-which-perl libimage-size-perl libio-string-perl libjson-xs-perl libwww-perl libparse-recdescent-perl liburi-perl libxml2 libxml-libxml-perl libxslt1.1 libxml-libxslt-perl texlive imagemagick perlmagick make'
+  }
+
+
+  exec { 'install latexml':
+    command => 'perl Makefile.PL && make && make install',
```

Comment:
What prevents this exec to run on every vagrant provision command? Normally you would have an creates=>